qmp/libvirtgo_linux.go

```diff
@@ -0,0 +1,327 @@
+// Copyright 2016 The go-qemu Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package qmp
+
+import (
+	"errors"
+	"log"
+	"os"
+	"sync"
+	"time"
+
+	libvirt "github.com/rgbkrk/libvirt-go"
+)
+
+// LibvirtGoEventsInterval constant used by the getPollInterval
+// method.
+const LibvirtGoEventsInterval = "LIBVIRTGO_EVENTS_INTERVAL"
```

Comment:
Remove this.  Adding configuration to a library with an environment variable is a code smell.